src/main/index.js

```diff
@@ -27,6 +27,13 @@ function runApp() {
         click: () => {
           browserWindow.webContents.send('showVideoStatistics', 'show')
         }
+      },
+      {
+        label: 'Open in a New Window',
+        visible: parameters.linkURL.includes('localhost'),
```

Comment:
Is this accurate enough?

Reply:
I think so, because any localhost link would be an in app page that the user might want to open in a new window.

Reply:
My only hesitation with this method is that we'd one day like to have a version of FreeTube for the web and going this route would basically make this feature non-existent in a web based version. If there's an alternative then I'd much rather go that route, but if there isn't anything then this is fine for now.

Reply:
a library like [this](https://github.com/johndatserakis/vue-simple-context-menu)(maybe see if there's a better alternative) would allow using a different context menu for both web and electron as it is a vue component. Bonus: using a vue library like this could fix the issue with #2479 in the future

Reply:
now I detect the address of the website

Reply:
@MarmadileManteater any thoughts on this